recipe/meta.yaml

```diff
@@ -9,9 +9,8 @@ source:
   sha256: 325f75eb80fbc5371136e37f323445309ca9f65b6c6f718d0d0e2189e5de1224
 
 build:
-  number: 1
+  number: 7
```

Comment:
Depending on the outcome of the discussion about `skimage/run-hessian.py`, this should either be reset to `1` or bumped to `2`.